testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/transaction/TransactionTestCase.java

```diff
@@ -146,11 +146,11 @@ public void testTransactionRequiredException() throws Exception {
 
 
     /**
-     * Tests JTA involving an EJB 3 SLSB which makes two DAO calls in transaction.
+     * Tests Jakarta Transactions involving an EJB 3 SLSB which makes two DAO calls in transaction.
```

Comment:
This search and replace has worked ;-) but most of them have resulted in incorrect grammar.

Reply:
:+1: please add `is` as in `Jakarta Transaction is`

Reply:
Hmm, `is` doesn't really fit.  How about changing to:
`Tests making two data access object calls in Jakarta Transactions transaction.`

Reply:
Or I suppose we could leave it as changed as well.

Reply:
okay then leaving as it is.